src/lib.rs

```diff
@@ -72,6 +71,34 @@ impl FormattedPersonnummer {
     }
 }
 
+/// [Seperator] is used by the short format to seperate birth date and serial information.
+/// A plus sign (+) indicates that the age is over 100 years.
+#[derive(Debug, Copy, Clone)]
+pub enum Seperator {
+    Plus,
+    Minus,
+}
+impl FromStr for Seperator {
+    type Err = &'static str;
```

Comment:
We already have `PersonnummerError` so we can use that one. Since this comes from the user input we can use `PersonnummerError::InvalidInput`.